### ISSUE TITLE
Feature: require GitHub Actions to be present

### ISSUE THREAD

**Issue by hazcod** [ossf/allstar#28]
Would be great if allstar could verify whether certain GitHub Actions exist and a minimum version.
Often GitHub Actions contain a set of security screenings to block the PR.

**Comment by jeffmendoza** [comment]
Yes, this is needed. Being able to generically configure an expected action/workflow to be present seems very useful. I believe defining how the config would work and the behavior of the policy will be tricky, help is needed here. For example, you can have multiple workflows in a repo, based on different triggers, and they may block/allow different things. I assume we would want to ensure a specific action is a part of a workflow that blocks something specific (PR merge, etc.).

**Comment by justaugustus** [comment]
This should be implemented now.
See https://github.com/ossf/allstar?tab=readme-ov-file#github-actions.
